### PR TITLE
Fix profile issue

### PR DESCRIPTION
**Pull request by PatrickTUy**
### PR Description
This PR fixed profile fetching.
### Description of tasks that were expected to be completed
Fix profile fetching
### Functionality
When users are logged in their profiles are displayed.
### How has this been tested?
checkout this deployed  [FE ](https://devpulse-review-app-135.herokuapp.com) review app
now login into the app and checkout the names of the logged in user
register a new user login as an admin or coordinator add them into a cohort and then login the new user, change their names through settings and login again as a coordinator or admin you will see their names reflecting the changes done.
### PR Checklist:
- [x] FIx profile fetching

### Track PR
N/A
### Screenshots (If appropriate)
N/A

### CODE REVIEW

**Comment by Procech20** [comment]
I deployed a review app for you to try out this pull request 👇🏽

Heroku App: https://devpulse-bn-review-app-40.herokuapp.com/


<hidden purpose="for-rewritable-pr-comment-action-use" value="Deploy"></hidden>

File: src/models/user.ts [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  role: {
+    type: String,
+    // type: mongoose.Types.ObjectId,
+    // ref: 'User',
```
Comment (Placide123) [inline src/models/user.ts]: can you remove this comment if there is no use for it in the future.

File: src/models/user.ts [hunk 2]
```diff
@@ -1,59 +1,60 @@
...
+    default: 'user',
+  },
+  // coordinator: {
+  //   type:mongoose.Types.ObjectId,
+  //   required: false,
+  //   ref: 'User'
+  //  },
```
Comment (Placide123) [inline src/models/user.ts]: and this comment can be removed to make clean code.

**Review by Placide123 — CHANGES_REQUESTED** [review]
Everything is working fine , but remove those commented code to make clean code.
and try to reduce commit you have made(target is to have one commit message but your PR has 3 commit thanks).

File: src/models/user.ts (hunk 1 above)
Comment (PatrickTUy) [reply src/models/user.ts]: Thanks @Placide123, it's now solved

File: src/models/user.ts (hunk 2 above)
Comment (PatrickTUy) [reply src/models/user.ts]: Done.

**Review by Placide123 — APPROVED** [review]

**Review by Tsa2341 — APPROVED** [review]

**Review by Procech20 — COMMENTED** [review]
Is it necessary to expose our _.env_ variables in the PR description

**Review by ceelogre — APPROVED** [review]

**Pull request merged**